test/unit/uk/gov/hmrc/testlogin/controllers/LoginControllerSpec.scala

```diff
@@ -52,12 +52,20 @@ class LoginControllerSpec extends UnitSpec with MockitoSugar with WithFakeApplic
   }
 
   "showLoginPage" should {
+
     "display the login page" in new Setup {
 
       val result = execute(underTest.showLoginPage("/continueUrl"))
 
       bodyOf(result) should include ("Sign in")
     }
+
+    "display the Create Test User link which opens a new browser-tab" in new Setup {
```

Comment:
This test just parses the HTML content.
Are we happy? Or we need also a Selenium test?